packages/p2p-v2/package.json

```diff
@@ -26,6 +26,7 @@
         "react-i18next": "^11.11.0",
         "react-modal": "^3.16.1",
         "react-router-dom": "^5.2.0",
+        "downshift": "^8.2.2",
```

Comment:
Is this only to add a combobox? if yes then you might need to request this from the deriv/ui members to help you and create one for you rather than using a third-party library.

Reply:
This is temporary @matin-deriv, I just added it, so that I'm not blocked. I also copied the dropdown from `Wallets`, until it's available in the `ui` library. These will all be removed